nacho/styles.css

```diff
@@ -0,0 +1,223 @@
+@import url('https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,100..900;1,100..900&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Jost', sans-serif;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f8f3e8;
+}
+
+header {
+    backdrop-filter: drop-shadow(4px 4px 10px blue);
+    background-color: #000;
+    background-image: url('img/header.webp');
+    background-size: cover;
+    background-position-y: center;
+    
+    color: #fff;
+    height: 300px;
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+}
+
+h1 {
+    padding-top: 100px;
+    text-align: center;
+    font-size: 3em;
+}
+
+h2 {
+    text-align: center;
+    font-size: 3em;
+    margin: 1em 0;
+    color: #0e233a;
+    font-weight: 500;
+}
+
+h3 {
+    color: #9c6d01;
+    font-size: 1.75em;
```

Comment:
I would have gone for `rem` instead of `em`. This is because if you have the div inside a parent div with a font-size, then this element will take 1.75 * font-size... meaning that different h3 could have different sizes

Reply:
Ok now I understand it, thank you!